src/QGCApplication.cc

```diff
@@ -647,8 +647,8 @@ QObject* QGCApplication::_rootQmlObject()
 
 void QGCApplication::showMessage(const QString& message)
 {
-    // Special case hack for ArduPilot prearm messages. These show up in the center of the map, so no need for popup.
-    if (message.contains("PreArm:")) {
+    // PreArm messages are handled by Vehicle and shown in Map
+    if (message.contains("PreArm")) {
```

Comment:
OR PREFLIGHT FAIL?
At the moment we also have a couple instances of "PREFLIGHT ADVICE".

Reply:
Oh, wait I forgot that was there. I'll change that as well. Which in turn means these should be shown with high severity and QGC wille at them.

Reply:
And I'll change to PREFLIGHT prefix only.